### PR TITLE
Grid simplified

### PR DESCRIPTION
**Pull request by krate27**
Screenshot (563)
Screenshot (564)

### CODE REVIEW

File: Components/grids.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    display: grid;
+    grid-template-columns: 30% 70%;
+    max-width: 65rem;
+    grid-auto-rows: minmax(150px, auto);
+    margin: 0 auto;
+    grid-gap: 2rem;  
+}
+@media screen and (max-width: 750px) {
+    .grid-wrapper {
+        margin-top: 30px;
```
Comment (AnuragSahu11) [inline Components/grids.css:48]: could have used rem

File: Components/grids.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    margin-top: 7rem;
+}
+.one{
+    background: #3bbced;
+    text-align: center;
+    padding: 2rem;
+    color: var(--white-color);
+}
+.two{
+    background: #777;
```
Comment (AnuragSahu11) [inline Components/grids.css]: could have used variable

**Review by AnuragSahu11 — APPROVED** [review]
Good work , good use of different colors, you can merge

File: Components/grids.css [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    grid-auto-rows: minmax(150px, auto);
+    margin: 0 auto;
+    grid-gap: 2rem;
+    
+}
+.grid-wrapper{
+    margin-top: 7rem;
+}
+.one{
+    background: #3bbced;
```
Comment (shuklahimanshu81) [inline Components/grids.css]: Use variable instead

File: Components/grids.css [hunk 4]
```diff
@@ -0,0 +1,50 @@
+.grid-50-50{
+    display: grid;
+    grid-template-columns: 50% 50%;
+    max-width: 65rem;
+    grid-auto-rows: minmax(150px, auto);
```
Comment (shuklahimanshu81) [inline Components/grids.css]: Using rem is better

File: Components/grids.css [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+}
+.grid-30-70{
+    display: grid;
+    grid-template-columns: 30% 70%;
+    max-width: 65rem;
+    grid-auto-rows: minmax(150px, auto);
+    margin: 0 auto;
+    grid-gap: 2rem;  
+}
+@media screen and (max-width: 750px) {
```
Comment (shuklahimanshu81) [inline Components/grids.css:46]: As suggested for these width for media screen you can create variable in your root file and then you can use them across the files

**Review by shuklahimanshu81 — APPROVED** [review]
Your code structure is clean and looks good. Green light from my side to merge.

File: Components/grids.css (hunk 2 above)
Comment (Kavita539) [inline Components/grids.css]: Suggestion : Use variables

File: Components/grids.css (hunk 1 above)
Comment (Kavita539) [inline Components/grids.css:48]: Suggestion : For more than 10 px use rem units.

**Review by Kavita539 — APPROVED** [review]
Other than that code is well structured and looks good.

File: Docsite-HTML-Files/grids.html [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+    <title> Docs | Grid-Simplified </title>
+    <link rel="icon" type="image/x-icon" href="/images/favicon.ico.png">
+    <script src="https://kit.fontawesome.com/c9ffd32afb.js" crossorigin="anonymous"></script>
+    <script src="https://kit.fontawesome.com/79dfc2e9eb.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="/Components/Villainess.css" />
+</head>
+
+<body>
+    <!-- Navbar  -->
+    <nav class="land-navbar" id="top-nav">
```
Comment (anand-tiwari) [inline Docsite-HTML-Files/grids.html:17]: I suggest to use class name based on BEM css concept

**Review by anand-tiwari — APPROVED** [review]
Added few comments related to css class name.
Use BEM while writing class.

Other looks good to me

**Pull request merged**